packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -154,6 +160,10 @@ class ConfigCommand extends FlutterCommand {
       _updateConfig('android-studio-dir', stringArg('android-studio-dir')!);
     }
 
+    if (argResults?.wasParsed('jdk-dir') ?? false) {
```

Comment:
I think here (and throughout this function) we should just cast `argResults` to a non-nullable value and get rid of all the nullable fallbacks.